app/javascript/components/server-components/ProductEditPage.tsx

```diff
@@ -244,7 +246,12 @@ const ProductEditPage = (props: Props) => {
 const ProductEditRouter = async (global: GlobalProps) => {
   const { router, context } = await buildStaticRouter(global, routes);
   const component = (props: Props) => (
-    <ProductEditContext.Provider value={createContextValue(props)}>
+    <ProductEditContext.Provider
```

Comment:
Would be nice to assert in controller spec as well this change is being record correctly in the DB when the params are passed.